src/main/scala/com/algorithmia/algo/Administrator.scala

```diff
@@ -60,16 +60,26 @@ class Administrator(client: AlgorithmiaClient){
     }
   }
 
-  def deleteOrganization(orgName:String): Boolean ={
-    val response = this.client.http.delete(url+s"/v1/organizations/${orgName}")
+  def getOrganizationTypeId(orgType: String): String = {
+    val response = this.client.http.get(url + s"/v1/organization/types")
+    if (response.code == 200) {
+      val orgTypes: List[OrganizationType] = Json.parse(response.body).as[List[OrganizationType]]
+      orgTypes.filter(o => o.name == orgType).head.id.toString
+    } else {
+      throw new IOException("Failed to get organization types, status code " + response.code)
+    }
```

Comment:
Do we want to reveal the status code here?

Reply:
yes, agree, makes sense to omit the status here as this is capturing the generic exception

Reply:
fixed